src/lib/paginator/paginator.html

```diff
@@ -1,75 +1,77 @@
-<div class="mat-paginator-container">
-  <div class="mat-paginator-page-size" *ngIf="!hidePageSize">
-    <div class="mat-paginator-page-size-label">
-      {{_intl.itemsPerPageLabel}}
-    </div>
-
-    <mat-form-field
-      *ngIf="_displayedPageSizeOptions.length > 1"
-      class="mat-paginator-page-size-select">
-      <mat-select
-        [value]="pageSize"
-        [aria-label]="_intl.itemsPerPageLabel"
-        (selectionChange)="_changePageSize($event.value)">
-        <mat-option *ngFor="let pageSizeOption of _displayedPageSizeOptions" [value]="pageSizeOption">
-          {{pageSizeOption}}
-        </mat-option>
-      </mat-select>
-    </mat-form-field>
+<div class="mat-paginator-wrapper">
```

Comment:
Any other possible name for this? I've never been a fan of "wrapper", since it doesn't convey anything about why it's there.

Reply:
I've renamed it to `mat-paginator-outer-container`.